### ISSUE TITLE
Old problem I learned to fix has "learned new tricks" and I can't figure out how to fix these.

### ISSUE THREAD

**Issue by Sidebottom1** [launchscout/nku#24]
![rubyrails error](https://f.cloud.github.com/assets/6433234/2047922/03ba32fa-8a32-11e3-988d-198f597c12cc.jpg)

Title is self-explanatory; I'm at my wit's end for now.  If more info is needed I can give it, but I assume my code is visible (to the professors at least).

**Comment by agilous** [comment]
Hmmm... You've got a lot going on here.

It looks like your UsersController is trying to find Makers but you're trying to iterate over a collection of Users. Look in the UsersController and see what the index method is doing. Are you assigning a collection to @users (as I would expect) or are you assigning to @makers (or something else)?

Then inside the Do loop beginning on line 13 you set the block variable to be "user" but you're calling "post" inside (lines 15 & 16). These should probably be something like "user.name" and "user.email" (maker.name, maker.email whichever you determined above).

Hope that helps.

**Comment by Sidebottom1** [comment]
Whoops--I thought I thoroughly combed my code for leftover fragments of the post code from the first assignment that I copied for this one.  I've fixed most of the initial errors, and I can now get to the user/student listing page, but create a new user is throwing this exception.  Am working on it now, but not sure if I can figure it out in short order.
![rubyrails error](https://f.cloud.github.com/assets/6433234/2052566/cfa2c632-8a98-11e3-8f26-8f91f8f360b8.jpg)
